lib/kdump_utils.pm

```diff
@@ -1,32 +1,27 @@
 # SUSE's openQA tests
 #
-# Copyright © 2016-2017 SUSE LLC
+# Copyright © 2017 SUSE LLC
 #
 # Copying and distribution of this file, with or without modification,
 # are permitted in any medium without royalty provided the copyright
 # notice and this notice are preserved.  This file is offered as-is,
 # without any warranty.
 
-# Summary: Run 'crash' utility on a kernel memory dump
-# Maintainer: Michal Nowak <mnowak@suse.com>
-
-use base "console_yasttest";
+package kdump_utils;
+use base Exporter;
+use Exporter;
 use strict;
 use testapi;
 use utils;
 
+our @EXPORT = qw(install_kernel_debuginfo prepare_for_kdump activate_kdump kdump_is_active do_kdump);
+
 sub install_kernel_debuginfo {
-    # list installed kernels
-    assert_script_run 'rpmquery -a kernel-default*';
-    # kernel debug symbols are huge, this can take a while
     script_run 'zypper ref';
-    zypper_call('-v in $(rpmquery kernel-default | sed "s/kernel-default/kernel-default-debuginfo/")');
```

Comment:
Can you keep versioned installation of kernel-default-debuginfo package, please? We used to have problems with wrong kernel-default-debuginfo versions in SUT's repositories, which led to weird issues with `crash`'es infrastructure.